### PR TITLE
fix image-classification import error with python3

### PR DESCRIPTION
**Pull request by yajiedesign**
fix import error with python3

### CODE REVIEW

File: example/image-classification/benchmark.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def validate(self, attrs):
                 sys.exit(1)
             try:
                 #check if the network exists
-                importlib.import_module('symbol.'+ args[0])
+                importlib.import_module('symbols.'+ args[0])
```
Comment (piiswrong) [inline example/image-classification/benchmark.py]: why rename to symbols?

File: example/image-classification/symbols/alexnet.py [hunk 2]
```diff
@@ -12,13 +12,13 @@ def get_symbol(num_classes, **kwargs):
         data=input_data, kernel=(11, 11), stride=(4, 4), num_filter=96)
     relu1 = mx.symbol.Activation(data=conv1, act_type="relu")
     pool1 = mx.symbol.Pooling(
-        data=relu1, pool_type="max", kernel=(3, 3), stride=(2,2))
+        data=relu1, pool_type="max", kernel=(3, 3), stride=(2,2), pad=(1, 1))
```
Comment (piiswrong) [inline example/image-classification/symbols/alexnet.py]: this breaks backward compatibility.

Comment (yajiedesign) [inline example/image-classification/symbols/alexnet.py]: yes.the pool1 no problem. no need for pad.
do other pool pad lead to compatibility issues?

File: example/image-classification/benchmark.py (hunk 1 above)
Comment (yajiedesign) [inline example/image-classification/benchmark.py]: because conflict with mx.symbol 
in python3 can't loading.

**Comment by mli** [comment]
alexnet and googlenet are wrongly switched #4826

**Pull request merged**